SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -7903,9 +7903,177 @@ void CEulerSolver::BC_Euler_Wall(CGeometry      *geometry,
                                  CConfig        *config,
                                  unsigned short val_marker) {
 
-  /*--- Call the equivalent symmetry plane boundary condition. ---*/
-  BC_Sym_Plane(geometry, solver_container, conv_numerics, visc_numerics, config, val_marker);
```

Comment:
At the moment, the Euler wall boundary condition is reverted from the symmetry plane condition in this draft PR as the previous Euler wall boundary condition gave the best verification against rigid motion as mentioned in Issue #828